tests/test_schema.py

```diff
@@ -1885,6 +1922,329 @@ def test_filter_non_exist_registry(self):
+                               email=None,
+                               is_bot=False,
+                               gender='M',
+                               country=None,
+                               individual=indv)
+        Identity.objects.create(uuid='B001',
+                                name='John Doe',
+                                email='jdoe@example.com',
+                                username='jdoe',
+                                source='scm',
+                                individual=indv)
+        Identity.objects.create(uuid='B002',
+                                name=None,
+                                email='jdoe@libresoft.es',
+                                username=None,
+                                source='scm',
+                                individual=indv)
+
+        timestamp_3 = datetime_utcnow()
+
+        # Tests
+
+        ts_1 = timestamp_1.strftime("%Y-%m-%dT%H:%M:%S.%f")
+        ts_2 = timestamp_2.strftime("%Y-%m-%dT%H:%M:%S.%f")
+        ts_3 = timestamp_3.strftime("%Y-%m-%dT%H:%M:%S.%f")
+
+        # Test individuals last_updated before first timestamp, it should return none
+        filter_no_indvs = '<={}'.format(ts_1)
+
+        client = graphene.test.Client(schema)
+        executed = client.execute(SH_INDIVIDUALS_LAST_UPDATED_FILTER % filter_no_indvs,
+                                  context_value=self.context_value)
+
+        individuals = executed['data']['individuals']['entities']
+        self.assertEqual(len(individuals), 0)
+
+        # Test individuals last_updated between first and second timestamp, it should return one
+        filter_first_indv = "{}..{}".format(ts_1, ts_2)
+
+        executed = client.execute(SH_INDIVIDUALS_LAST_UPDATED_FILTER % filter_first_indv,
+                                  context_value=self.context_value)
+
+        individuals = executed['data']['individuals']['entities']
+        self.assertEqual(len(individuals), 1)
+
+        # Test John Smith individual
+        indv = individuals[0]
+        self.assertEqual(indv['mk'], 'a9b403e150dd4af8953a52a4bb841051e4b705d9')
+
+        self.assertEqual(indv['profile']['name'], None)
+        self.assertEqual(indv['profile']['email'], 'jsmith@example.com')
+        self.assertEqual(indv['profile']['isBot'], True)
+        self.assertEqual(indv['profile']['country']['code'], 'US')
+        self.assertEqual(indv['profile']['country']['name'], 'United States of America')
+
+        identities = indv['identities']
+        identities.sort(key=lambda x: x['uuid'])
+        self.assertEqual(len(identities), 3)
+
+        id1 = identities[0]
+        self.assertEqual(id1['email'], 'jsmith@example.com')
+
+        id2 = identities[1]
+        self.assertEqual(id2['email'], 'jsmith@bitergia.com')
+        self.assertEqual(id2['source'], 'scm')
+
+        id3 = identities[2]
+        self.assertEqual(id3['email'], 'jsmith@bitergia.com')
+        self.assertEqual(id3['source'], 'mls')
```

Comment:
You can remove this. I don't think we need to check this. Checking the `mk` only should be enough.